aws/validators.go

```diff
@@ -861,6 +861,16 @@ func validateAwsEcsPlacementStrategy(stratType, stratField string) error {
 	return nil
 }
 
+func validateAwsEmrBidPrice(v interface{}, k string) (ws []string, errors []error) {
+	value := v.(string)
+
+	if len(value) > 256 {
+		errors = append(errors, fmt.Errorf("%q cannot be longer than 256 characters", k))
+	}
```

Comment:
You should also check for the `[\u0020-\uD7FF\uE000-\uFFFD\uD800\uDC00-\uDBFF\uDFFF\r\n\t]*` pattern validation.

Also, this will need to have a unit test attached. You can find many examples in the `validators_test.go` file